UE/Tests/Application/Mocks/ISmsDbMock.hpp

```diff
@@ -2,6 +2,7 @@
 
 #include <gmock/gmock.h>
 
+#include "Sms.hpp"
```

Comment:
try to use forward declaration

Reply:
I tried  `MOCK_METHOD(const SmsMessages&, getSmsMessages, (), (final));`  told me that in template field has incomplete type 'ue::Sms'